meetings/summaries/2025.md

```diff
@@ -0,0 +1,17 @@
+## 2025 Technical Committee Summary
+
+
+2025 has been a decisive year for the Technical Committee (TC), marked by strengthened governance, a more mature incubation framework, and steady growth in the OpenRail project ecosystem.
+
+The election of **Florian** as chair of the technical committee, **first representative of a project** demonstrating the maturity of the technical committee's governance.
+
+The TC completed and formalized both Stage 2 and Stage 3 incubation criteria, enabling more consistent project evaluation. Incubation stages were renamed to **Onboarded → Qualified → Adopted**, providing clarity for members, contributors, and external partners. We published a breakdown and description of the capabilities of the railway system to help openrail projects and future users to meet up.
```

Comment:
I think it would be good to mention that we plan to make intense use of the capabilities to map projects and project opportunities moving forward.